daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/WarningLog.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.speedy
+
+import org.slf4j.Logger
+import scala.collection.mutable.ArrayBuffer
+
+private[lf] final case class WarningLog (logger: Logger) {
```

Comment:
I dislike using `case class` for not mutable data. 
Since you instantiate the object only once, I think you can remove the `case` keyword here  and use a `new` there.

Reply:
Done, thanks!